src/components/Table/index.tsx

```diff
@@ -232,22 +232,25 @@ const Table: React.FC<PropsWithChildren<ITable>> = ({
       )}
       <ContainerView ref={tableRef}>
         <TableBody smaller={smaller}>
-          {!isMobile && !isTablet && (
-            <TableRow>
-              {header?.map((item, index) => (
-                <HeaderItem
-                  key={JSON.stringify(item)}
-                  smaller={smaller}
-                  totalColumns={header.length}
-                  currentColumn={index}
-                  dynamicWidth={rowSections(item)?.[index]?.width}
-                  maxWidth={rowSections(item)?.[index]?.maxWidth}
-                >
-                  {item}
-                </HeaderItem>
-              ))}
-            </TableRow>
-          )}
+          {!isMobile &&
+            !isTablet &&
+            !response?.items &&
+            response?.items?.length === 0 && (
```

Comment:
should show header if there are items
![image](https://github.com/user-attachments/assets/f8eb6166-0e3b-486f-9017-8927c4f9cfb2)